### DISCUSSION TITLE
Will Bevy evolve to look like Godot engine Visual editor ?

### DISCUSSION THREAD

**Discussion by pyweeker** [bevyengine/bevy#4201] [Ideas]
Is it planned for long term to get some tools built upon Bevy to speed up and making easier for beginners making games ?

**Comment by alice-i-cecile** [comment]
Yes, that's the long-term plan still: see #85. The strategy is to build it with bevy_ui (much like Godot did), allowing us to evolve the capabilities of the UI as we develop the editor itself.

**Comment by pyweeker** [comment]
Does Fyrox / rg3d represent a competitor ?
